### DISCUSSION TITLE
Add object count for every plate or selected objects

### DISCUSSION THREAD

**Discussion by VDechev** [OrcaSlicer/OrcaSlicer#2715] [Ideas]
Hello, recently I started printing a lot of small models and it was hard to keep how many of them are on the plate. 

I would love to see something similar to Windows where it shows how many files are selected(in our case 3d models)
![image](https://github.com/SoftFever/OrcaSlicer/assets/41317635/526e8910-1c89-475f-aeb7-b6034f5728bb)

Another similar idea suggestion is to add the object count somewhere next to each plate

**Comment by rkarlsba** [comment]
I'd voach for this! I'm also with a current project with >200 objects, but I have no idea of how many there are in total or per plate, since there is no counter. Please add this - it shouldn't be hard.

**Comment by iyadmn** [comment]
Yes, I agree with the proposal. When you select the "fill the bed with copies" option, you don't know how many copies will be printed in total on that tray.
